### DISCUSSION TITLE
Time-based scoring

### DISCUSSION THREAD

**Discussion by mapeje** [mawoka-myblock/ClassQuiz#311] [Ideas]
Would it be possible to get more points by answering faster, much like Kahoot or Menti? This would increase the engagement and also easier to announce a winner if many participants end at the same result. Unfortunately, I have zero coding skills so I cannot really contribute to the project more than ideas and feedback.

**Comment by TheNomad11** [comment]
Yes, I agree, this would be very useful. Then Classquiz really could compete with Kahoot
